fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,19 @@
 */
 int fizzbuzz(int n)
 {
-
+   int counter = 0;
+   for (int i = 1; i <= n; i++) 
+   {
+       if (i % 15 == 0)
+        printf("%d Fizzbuzz\n", i);
+    else if (i % 3 == 0)
+        printf("%d Fizz\n", i);
+        else if (i % 5 == 0)
+        printf("%d Buzz\n", i);
+        else
+            counter = counter +1;
+   }
+   return counter;
 }
```

Comment:
while this works, many developers prefer to use curly braces because it is less error-prone. For example, (maybe you know this already, but in case you didn't), if you and a second statement to an if/else if/ else block without braces, you will get an error. Others would also add that curly braces make the code more readable in C.